doc/password.md

```diff
@@ -0,0 +1,8 @@
+Password
+=======
+
+The `password` option in `config.json` allows you to set which algorythm we use to hash a value. Primarily this is being used for `accountId` when stored so we aren't using the original `accountId`.
+
+    "password": {
+        "hashAlgo": "sha256"
+    }
```

Comment:
Would you add a bit about being able to find available hashing methods in `lib/ciphers-and-hashes.js`?